.circleci/config.yml

```diff
@@ -63,8 +63,8 @@ jobs:
             - vendor/bundle
 
       - run:
-          name: rails db:setup
-          command: bundle exec bin/rails db:wait db:setup
+          name: Set up the DB
+          command: bundle exec bin/rails db:wait db:reset
```

Comment:
why changed this? reset would try to remove existing database first.

Reply:
The rest looks nice!

Reply:
I thought that that would be an easy way to resolve the "migrations are pending" error :grimacing: But it's not.
Adding `db:migrate` afterwards didn't help either :thinking: 

Ideas?

Reply:
if the schema file is updated, then `db:setup` should apply it an no migrations should be missing
Maybe schema file version is wrong for some reason?

Reply:
Well, zync uses `structure.sql` instead of schema, and to be honest, I have no idea how it works and how it is related to migrations. 

And actually, I think there's something weird done with the filename (adding `-12` for any PG version > 11), I think I'll be fixing it in another PR.

Reply:
It should be the content of `schema_migrations` table. Maybe we should drop support for postgres < v12 at some point.

Reply:
Yes, definitely, but well, first we need to migrate SaaS to at least 13, and upgrade 3scale-operator to use 13 as well (it uses 10 now)